skyportal/plot.py

```diff
@@ -1345,7 +1345,13 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
     else:
         # tabs for mag, flux
         tabs = Tabs(tabs=[p2, p1], width=width, height=height + 90, sizing_mode='fixed')
-    return bokeh_embed.json_item(tabs)
+
+    try:
+        return bokeh_embed.json_item(tabs)
+    except ValueError:
+        with pd.option_context('display.max_rows', None, 'display.max_columns', None):
+            print(data)
```

Comment:
Maybe some context here so we know what we are seeing in the log?  Otherwise the PR looks good, thanks!

Reply:
added another print statement.